engine/tests/settings.cpp

```diff
@@ -52,12 +52,21 @@ TEST_CASE("cubos::engine::settingsPlugin")
     };
 
     EmbeddedArchive::registerData("settings.json", settingsData);
-    FileSystem::mount("/settings.json", std::make_unique<EmbeddedArchive>("settings.json"));
+
+    if (!FileSystem::mount("/settings.json", std::make_unique<EmbeddedArchive>("settings.json")))
+    {
+        FileSystem::unmount("/settings.json");
+        FileSystem::mount("/settings.json", std::make_unique<EmbeddedArchive>("settings.json"));
+    }
 
     // Run the actual test.
     Cubos cubos{argc, argv};
     cubos.plugin(settingsPlugin);
 
+    cubos.startupSystem("configure Assets").tagged(settingsTag).call([](Settings& settings) {
+        settings.setBool("assets.io.enable", false);
+    });
```

Comment:
Why is this here?

Reply:
Don't mark stuff as resolved without answering them! Otherwise I might just forget about it.
This test doesn't even use assets so it really shouldn't need this